src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
+    [InlineData(WatcherChangeTypes.Created, "created", false)]
+    [InlineData(WatcherChangeTypes.Deleted, "deleted", false)]
+    [InlineData(WatcherChangeTypes.Deleted, "changed", false)]
+    [InlineData(WatcherChangeTypes.Renamed, "renamed", true)]
+    [InlineData(WatcherChangeTypes.Renamed, "changed", true)]
```

Comment:
Could you please double-check these?  I don't understand, for example, what it means in this case to have Deleted + "changed".  The name is only used as part of the assert (and filename) as far as I can see, rather than being something that's actually tested.  If it's necessary, fine, but I'm not seeing it.  The old tests may just have been bogus in this regard.